### PR TITLE
Add translations for Imperative Config for Bahasa Indonesia

### PR DESCRIPTION
**Pull request by qbl**
This PR addresses [#13929](https://github.com/kubernetes/website/issues/13929).

### CODE REVIEW

File: content/id/docs/concepts/overview/object-management-kubectl/imperative-config.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+* Konfigurasi objek imperatif
+* Konfigurasi objek deklaratif
+
+Lihat [Pengelolaan Objek Kubernetes](/id/docs/concepts/overview/object-management-kubectl/overview/) untuk mengenali lebih lanjut kelebihan dan kekurangan dari tiap cara pengelolaan objek.
+
+## Cara membuat objek
+
+Kamu bisa menggunakan perintah `kubectl create -f` untuk membuat sebuah objek dari sebuah file konfigurasi. Rujuk dokumen [referensi API Kubernetes](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/) untuk detil lebih lanjut.
+
+- `kubectl create -f <filename|url>`
```
Comment (sftim) [inline content/id/docs/concepts/overview/object-management-kubectl/imperative-config.md]: “filename” in English; is this correct?

File: content/id/docs/concepts/overview/object-management-kubectl/imperative-config.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    labels:
+      controller-selector: "extensions/v1beta1/deployment/nginx"
+```
+
+{{% /capture %}}
+
+{{% capture whatsnext %}}
+- [Pengelolaan Objek Kubernetes Menggunakan Perintah Imperatif](/docs/concepts/overview/object-management-kubectl/imperative-command/)
+- [Pengelolaan Objek Kubernetes secara Deklaratif dengan Menggunakan File Konfigurasi](/docs/concepts/overview/object-management-kubectl/declarative-config/)
+- [Kubectl Command Reference](/docs/reference/generated/kubectl/kubectl/)
```
Comment (sftim) [inline content/id/docs/concepts/overview/object-management-kubectl/imperative-config.md]: “Kubectl Command Reference” is in English; is that intentional?

File: content/id/docs/concepts/overview/object-management-kubectl/imperative-config.md (hunk 1 above)
Comment (qbl) [reply content/id/docs/concepts/overview/object-management-kubectl/imperative-config.md]: hi @sftim, you're right, I missed translating that part. I have updated it in my new commit.

File: content/id/docs/concepts/overview/object-management-kubectl/imperative-config.md (hunk 2 above)
Comment (qbl) [reply content/id/docs/concepts/overview/object-management-kubectl/imperative-config.md]: hi @sftim, it's unintentional, I missed translating that part. I have updated it in my new commit.

Comment (sftim) [reply content/id/docs/concepts/overview/object-management-kubectl/imperative-config.md]: Thanks for clarifying & fixing!

**Comment by sftim** [comment]
@qbl you might want to [squash](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History#_squashing) those commits so this is ready to review & merge (per https://kubernetes.io/docs/contribute/localization/#branching-strategy)

**Pull request merged**